test/components/formcomponent.test.js

```diff
@@ -112,6 +112,12 @@ lab.experiment('FormComponent', () => {
       expect(viewModel.label).to.exist()
       expect(viewModel.label.text).to.equal('testFormComponent')
     })
+    lab.test('with empty title', () => {
+      const formComponent = new FormComponent({ name: componentName, title: '' })
+      const viewModel = formComponent.getViewModel({}, formData)
+      expect(viewModel.label).to.exist()
+      expect(viewModel.label.text).to.equal('')
+    })
```

Comment:
As above, I don't think that titles should be empty - it may be worth having a quick chat about this in case there is a genuine use case and I'm just blissfully unaware of it